### PR TITLE
Adding Github Actions to Cellveyor and an image folder

### PR DESCRIPTION
**Pull request by KevenDuverglas**
This will allow celllveyor to have its pretty little green checkmark

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,24 @@
 
 ## Example Command
 
+[![build](https://github.com/KevenDuverglas/keven-cellveyor/actions/workflows/build.yml/badge.svg)](https://https://github.com/KevenDuverglas/keven-cellveyor/actions/workflows/build.yml)
```
Comment (gkapfham) [inline README.md:5]: Hi @KevenDuverglas, you need to make sure that the documentation does not link to a badge that is inside of a repository in your account.

File: README.md [hunk 2]
```diff
@@ -2,16 +2,24 @@
...
+[![Language:Python](https://img.shields.io/badge/Language-Python-blue.svg)](https://github.com/KevenDuverglas/keven-cellveyor/search?l=python)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-Yes-blue.svg)](https://github.com/KevenDuverglas/keven-cellveyor/graphs/commit-activity)
+
 ```
 poetry run cellveyor --spreadsheet-directory \
-/home/gkapfham/working/data/gradebook/2023 --spreadsheet-file CMPSC-203-Fall-2023-Gradebook.xlsx \
+/home/gkapfham/working/data/gradebook/2023 --spreadsheet-file
+CMPSC-203-Fall-2023-Gradebook.xlsx \
```
Comment (gkapfham) [inline README.md:12]: This should be further generalized.

It should not, ultimately, have anything to do with our class.

File: README.md [hunk 3]
```diff
@@ -2,16 +2,24 @@
...
 --sheet-name Main \
 --key-attribute "Student GitHub" \
 --key-value "gkapfham" \
 --column-regexp "^(Summary Grade|Final Grade) .*$" \
 --feedback-regexp "Summary Grade 1 - Feedback" \
---feedback-file /home/gkapfham/working/teaching/github-classroom/feedback/all/feedback.yml \
---feedback-file /home/gkapfham/working/teaching/github-classroom/feedback/developer-development/feedback-overall-course-assessment.yml \
+--feedback-file /home/gkapfham/working/teaching/github-classroom/
```
Comment (gkapfham) [inline README.md:18]: All of these paths need to be generalize and clarified.

**Review by gkapfham — CHANGES_REQUESTED** [review]
Hello @KevenDuverglas, thanks for creating this PR!

Please note that several parts of the documentation need to be generalized further so that the command it not customized to either a specific directory structure or to specific spreadsheets.

It is also worth noting that you are proposing to add a wide variety of binary files to this repository. We cannot allow these type of derived files into the repository. Please chat with Sabrina as she can explain how these files got into the repository in the first place and what we can do to prevent this from happening in the future.

**Review by jnormile — CHANGES_REQUESTED** [review]
@KevenDuverglas, I'm still seeing a lot of derived binary files (like the ones that end with a `.pyc` extension, for example). Whenever you get a chance, please remove these from the PR--thanks!

**Review by jnormile — APPROVED** [review]
Great work putting the GH Actions together, and thanks for taking the time to start us off with a passing build!

Reach out to your peers to get the required reviews--and ask @gkapfham to update his review since he requested changes--and then we'll be good to go! Thanks!

**Review by hayleepierce — APPROVED** [review]
This looks great! Thank you for all your work on this issue!

**Review by gkapfham — APPROVED** [review]
Hello @KevenDuverglas and @jnormile, I am willing to approve this PR and see it merged into our project. With that said, @KevenDuverglas, there are two points that I wanted to raise after discussing them with @jnormile.

1. It is generally not a good idea to have a top-level directory called `spreadsheets` that is not essential for the use of this tool. I think that it would be better if this directory was either a "dot directory" that was hidden or if it was, for instance, inside of the `.github` directory or in a special `testing` directory.
2. It is also important to point out that right now there is a `feedback.yml` file that is inside of the `spreadsheets` directory which does not make sense because of the fact that a YML file is not a spreadsheet.

With that said, perhaps we can agree to overlook those issues right now so that we can merge this content and then revise things later as needed.

**Pull request merged**